chapters/introduction.tex

```diff
@@ -97,59 +77,74 @@ \section{Outline of this book}
 handling data processing and analytical tasks.
 
 
-\section{Adopting reproducible workflows}
+\section{Adopting reproducible tools}
 We will provide free, open-source, and platform-agnostic tools wherever possible,
 and point to more detailed instructions when relevant.
-Stata is the notable exception here due to its current popularity in development economics.
 Most tools have a learning and adaptation process,
 meaning you will become most comfortable with each tool
 only by using it in real-world work.
 Get to know them well early on,
 so that you do not spend a lot of time learning through trial and error.
 
-While adopting the workflows and mindsets described in this book requires an up-front cost,
-it will save you (and your collaborators) a lot of time and hassle very quickly.
-In part this is because you will learn how to implement essential practices directly;
-in part because you will find tools for the more advanced practices;
-and most importantly because you will acquire the mindset of doing research with a high-quality data focus.
-We hope you will find this book helpful for accomplishing all of the above,
-and that mastery of data helps you make an impact!
+Stata is the notable exception here due to its current popularity in development economics.
+We focus on Stata-specific tools and instructions in this book.
+Hence, we will use the terms ``script'' and ``do-file''
+interchangeably to refer to Stata code throughout.
+Stata is primarily a scripting language for statistics and data,
+meaning that its users often come from economics and statistics backgrounds
+and understand Stata to be encoding a set of tasks as a record for the future.
+We believe that this must change somewhat:
+in particular, we think that practitioners of Stata
+must begin to think about their code and programming workflows
+just as methodologically as they think about their research workflows,
+and that people who adopt this approach will be dramatically
+more capable in their analytical ability.
+This means that they will be more productive when managing teams,
+and more able to focus on the challenges of experimental design
+and econometric analysis, rather than spending excessive time
+re-solving problems on the computer.
+To support this goal, this book also includes
+an introductory Stata Style Guide
+that we use in our work, which provides
+some new standards for coding so that code styles
+can be harmonized across teams for easier understanding and reuse of code.
+Stata has relatively few resources of this type available,
+and the ones that we have created and shared here
+we hope will be an asset to all its users.
```

Comment:
Was this caused by merging different branches? I'm pretty sure this section was heavily edited in the last version of the intro I read.

Reply:
2e25f2ca1bb6b522023a56f58f99d55a57c2b181 , among the other comments